app/controllers/idv_controller.rb

```diff
@@ -24,6 +26,8 @@ def index
     end
   end
 
+  def show; end
```

Comment:
Question: Do we want this here, or shall we create a new controller for the new flow?

Overall, my expectation is that we'd only need a single controller, single action for every step in the flow. In early phases, we may need to have a way to initialize the React app with some prepopulated data.

Reply:
+1 from me

Reply:
Moved to new `VerifyController` in 563667992.